src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    @Override
+    public String getReport(StringBuilder reportBuilder, Map<String, Integer> fruitMap) {
```

Comment:
```suggestion
    public String getReport( Map<String, Integer> fruitMap) {
```
create builder inside method

Reply:
fixed